ue-task-management/src/universal-editor-ui-1/web-src/src/components/ExtensionRegistration.js

```diff
@@ -0,0 +1,38 @@
+/*
+ * <license header>
+ */
+
+import { useEffect } from "react";
+import { Text } from "@adobe/react-spectrum";
+import { extensionId } from "./Constants";
+import { register } from "@adobe/uix-guest";
+
+function ExtensionRegistration() {
+  useEffect(() => {
+    const init = async () => {
+      const registrationConfig = {
+        id: extensionId,
+        methods: {
+          rightPanel: {
+            addRails() {
+              return [
+                {
+                  id: "taskManagement",
+                  header: "Task Management",
+                  url: "https://experience-qa.adobe.com/solutions/aem-sites-reverie-sites-workfront-integration/static-assets/resources/embed.html?route=TaskManagementWidget",
```

Comment:
I think it makes sense to use the production URL.

Reply:
As an idea. you can use it as a variable from the .env to specify if you want to run app on stage or prod environment

Reply:
do you mean to inject it at runtime? from what I've seen in other extensions in this repo, env files are ignored from source control, so we would have to provide it, I left it hardcoded to production url for now.